api/app/Http/Controllers/BaseController.php

```diff
@@ -153,68 +157,4 @@ public function deleteMany(Request $request)
 
         return response(null, 204);
     }
```

Comment:
I believe we can do that with the help of Model::push()
Need to test that.

In general we should be able to do the things like this

BelongsTo:
$articleModel->setAuthor($authorModel);
$articleRepository->save($articleModel);

HasMany or ManyMany
$articleModel->addAuthor($authorModel);
$articleRepository->save($articleModel);